src/action.js

```diff
@@ -115,7 +116,10 @@ function validateTitle(pullRequest) {
 function validateBody(pullRequest) {
   core.info("Validating body..");
   const { body } = pullRequest;
-  if (!body) {
+  if (
+    !body &&
+    !JSON.parse(core.getInput("generate_release_notes") || false) === true
```

Comment:
Maybe like this instead?
```suggestion
    JSON.parse(core.getInput("generate_release_notes") || false) === false
```

Reply:
Ah, yes. It was done in this roundabout way to stay consistent with other places where input values are checked, but this does make it more confusing than it needs to be.